src/Command.js

```diff
@@ -70,6 +78,11 @@ export default class Command {
     log.pause();
     log.heading = "lerna";
 
+    // Default to silent log level for JSON output
+    if (flags.json) {
```

Comment:
This isn't necessary, actually, as all logging is emitted to `stderr` and only the JSON string is emitted to `stdout`. When you pipe the output, it only includes `stdout` (unless you do fancy stdio redirects, of course).

It's too bad I didn't write a `lerna ls` integration test that could demonstrate this :/